.github/workflows/codeception.yml

```
@@ -0,0 +1,78 @@
+name: "Codeception Tests"
+
+on:
+    schedule:
+        -   cron: '0 3 * * 1,3,5'
+    pull_request:
+        branches:
+            - "[0-9]+.[0-9]+"
+            - "[0-9]+.x"
+    push:
+        branches:
+            - "[0-9]+.[0-9]+"
+            - "[0-9]+.x"
+            - "*_actions"
```

```suggestion
on:
    schedule:
        - cron: '0 3 * * 1,3,5'
    pull_request:
        branches:
            - "[0-9]+.[0-9]+"
            - "[0-9]+.x"
        paths-ignore:
            - 'doc/**'
            - 'src/Resources/public/**'
    push:
        branches:
            - "[0-9]+.[0-9]+"
            - "[0-9]+.x"
        paths-ignore:
            - 'doc/**'
            - 'src/Resources/public/**'
```

probably this rules set (from [datahub](https://github.com/pimcore/data-hub/blob/d9ba7060874381b5c0e9b9c8479ce8f70b734ea1/.github/workflows/codeception.yml#L4-L20)) are more up-to-date, it would skip PR with only docs and stuff in public (js,css) changes